docs/book/Makefile

```
@@ -20,30 +20,45 @@ MDBOOK := $(TOOLS_BIN_DIR)/mdbook
 MDBOOK_EMBED := $(TOOLS_BIN_DIR)/mdbook-embed
 MDBOOK_RELEASELINK := $(TOOLS_BIN_DIR)/mdbook-releaselink
 MDBOOK_TABULATE := $(TOOLS_BIN_DIR)/mdbook-tabulate
+GENCRDAPIREFERENCEDOCS := $(TOOLS_BIN_DIR)/gen-crd-api-reference-docs
+GENCRDAPIREFERENCEDOCS_SRCS := $(call rwildcard,.,gen-crd-api-reference-docs/*.*)
 BOOK_SRCS :=  $(call rwildcard,.,*.*)
+API_SRCS := $(call rwildcard,.,../../cmd/clusterawsadm/api) $(call rwildcard,.,../../api)
 
 OS := $(shell go env GOOS)
 ARCH := $(shell go env GOARCH)
 PATH := $(abspath $(TOOLS_BIN_DIR)):$(PATH)
 export PATH
 
+src/crd:
+	mkdir -p src/crd
+
+src/crd/cluster-api-aws.md: $(GENCRDAPIREFERENCEDOCS) src/crd $(API_SRCS) $(GENCRDAPIREFERENCEDOCS_SRCS) $(BOOK_SRCS)
+	$(GENCRDAPIREFERENCEDOCS) -template-dir gen-crd-api-reference-docs/template -config "gen-crd-api-reference-docs/config.json" \
+		-api-dir "../../api/v1alpha3" \
+		-out-file $@
+
+src/crd/clusterawsadm.md: $(GENCRDAPIREFERENCEDOCS) src/crd $(API_SRCS) $(GENCRDAPIREFERENCEDOCS_SRCS) $(BOOK_SRCS)
```

Do we need API_SRCS and BOOK_SRCS? 
```suggestion
src/crd/clusterawsadm.md: $(GENCRDAPIREFERENCEDOCS) src/crd $(GENCRDAPIREFERENCEDOCS_SRCS) 
```
```suggestion
src/crd/clusterawsadm.md: $(GENCRDAPIREFERENCEDOCS) src/crd $(API_SRCS) $(GENCRDAPIREFERENCEDOCS_SRCS) $(BOOK_SRCS)
```

---

We do because we want the CRD docs to update when the API changes.